HW 5 (mock-data + JS)/Task 2/JS/index.js

```diff
@@ -0,0 +1,7 @@
+const arr = ['The', 'quick', 'brown', 'fox', 'jumps', 'over', 'the', 'lazy', 'dog'];
+
+function join(array) {
```

Comment:
Функція повинна приймати розділювач як другий аргумент.
```suggestion
function join(array, delimiter = ' ') {
```